### PR TITLE
Fix module_parent_name not present on rails < 6.0.0

### LINKED ISSUE #95: [Bug] Fix `module_parent_name` not available on rails < 6
## Bug report:

According to [apidock](https://apidock.com/rails/v6.0.0/Module/module_parent_name), `module_parent_name` is only available on rails >= 6.0.0. With the change introduced on #93 we wouldn't be compatible with rails 5, which we aim to be.

The solution would be to use `module_parent_name` whenever possible and fallback to [`parent_name`](https://apidock.com/rails/v6.0.0/Module/parent_name) in other cases

### PR DESCRIPTION
**Pull request by Snick555**
Fixes #95

### CODE REVIEW

File: app/helpers/exception_hunter/application_helper.rb [hunk 1]
```diff
@@ -1,5 +1,13 @@
 module ExceptionHunter
   module ApplicationHelper
     include Pagy::Frontend
+
+    def module_parent_name
```
Comment (brunvez) [inline app/helpers/exception_hunter/application_helper.rb:5]:
```suggestion
    def application_name
```

File: app/views/layouts/exception_hunter/application.html.erb [hunk 2]
```diff
@@ -25,7 +25,7 @@
         <div class="column column-40">
           <%= link_to errors_path do %>
             <div class="nav__title">
-              Exception Hunter / <%= Rails.application.class.module_parent_name %>
+              Exception Hunter / <%= module_parent_name %>
```
Comment (brunvez) [inline app/views/layouts/exception_hunter/application.html.erb:28]:
```suggestion
              Exception Hunter / <%= application_name %>
```

File: app/helpers/exception_hunter/application_helper.rb (hunk 1 above)
Comment (brunvez) [reply app/helpers/exception_hunter/application_helper.rb:5]: Do you mind doing a straightforward test just as documentation?

**Comment by Snick555** [comment]
@brunvez Thanks for your comments. Updated.

**Review by brunvez — APPROVED** [review]
Thank you for the PR! Go for that t-shirt!

**Pull request merged**